bundles/http/Citrix/Zoom-Meetings/e40510b3-a760-43b4-ab16-56572dc6303b.js

```diff
@@ -39,19 +42,24 @@ async function syncMeetings(dataStore, client, users) {
     let next_page_token="";
     const page_size=200;
     let meetingids = new Set();
+    
+    if(users.length === 0){
+        return meetingids
+    }
+
     do {
         const response = await client.fetch(`users/${users[i].userid}/meetings?type=upcoming&next_page_token=${next_page_token}&page_size=${page_size}`);
         const json = await response.json();
         if (!(response.ok)) {
-            throw new Error(json);
+            throw new Error(JSON.stringify(json));
         }
         next_page_token = json.next_page_token ?? "";
         const data = json.meetings;
         let meetingdata = data.filter(meeting =>{
             return meeting.type == 2 || meeting.type == 8 
         }).map(meeting => {
             if(!meetingids.has(meeting.id)){
```

Comment:
this won't work because `meetingids` are no longer list of id but list of meetings and numer compared with object will be always `false`. If you need unique list of meetings then do 1) mapping 2) make the list unique using (build-in) lodash library (`const _ = library.load('lodash')` and check lodash docs for uniqueBy